crypto-ffi/src/generic/mod.rs

```diff
@@ -1130,7 +1130,7 @@ impl CoreCrypto {
             .map_err(CoreCryptoError::generic())?;
         let configuration = MlsCentralConfiguration::try_new(
             path,
-            key,
+            key.0.clone(),
```

Comment:
Instead of explicit `.0`, maybe `#[derive(derive_more::Deref)]` on the local `DatabaseKey` so cloning produces the inner type?

Should produce a smaller diff when we can eliminate the local wrapper.